Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/SpatialSoundSettings.cs

```diff
@@ -40,10 +41,7 @@ public static class SpatialSoundSettings
         /// </summary>
         private enum SpatialSoundParameters
         {
-            RoomSize = 1,
-            MinGain,
-            MaxGain,
-            UnityGainDistance
+            RoomSize = 1
```

Comment:
How will this affect people already using the parameters?

Will they get out of index exceptions?

Reply:
It shouldn't, as these values map to the parameter indices in the spatializer plugin (in fact, they wind up cast as an int when calling into the plugin).

Reply:
We should be safe, this enum is private to the class, and the functions that used them are now marked obsolete and have no code in them.